hw/ip/spi_device/doc/programmers_guide.md

```diff
@@ -53,11 +54,12 @@ Due to the CDC issue, the SW is only permitted to update the registers when the
 ### TPM Write
 
 1. The host system sends the TPM write command with the address.
-1. The TPM submodule pushes the command and the address to the TPM_CMD_ADDR CSR.
-1. The TPM submodule checks the write FIFO status.
-1. If not empty, the TPM submodule sends `WAIT` to the host system.
-1. When the FIFO is empty, the TPM sends `START` to the host system, receives the payload, and stores the data into the write FIFO.
-1. The SW, in the meantime, reads TPM_CMD_ADDR then reads the write FIFO data when the FIFO is available.
+1. The TPM submodule checks the write FIFO and cmdaddr FIFO statuses.
```

Comment:
Can you fix the enumeration? Although the markdown render can enumerate it, It's also nice to be able to read the doc from the github repo.

Not related, but I miss such description for Flash mode.

Reply:
@jesultra can you take a look at this also to make sure the interface is aligned with your usecase?